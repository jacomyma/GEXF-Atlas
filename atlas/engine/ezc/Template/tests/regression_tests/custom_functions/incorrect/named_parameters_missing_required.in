{named_parameters(p3 ="b", p2="a")}
