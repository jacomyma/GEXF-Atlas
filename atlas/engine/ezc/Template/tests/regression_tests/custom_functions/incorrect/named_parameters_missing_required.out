mock:1:2: Parameter 1 is not given and has not a default value.

{named_parameters(p3 ="b", p2="
 ^
